provider-management.md

```diff
@@ -0,0 +1,34 @@
+# Provider Management
+
+![Provider Manager Dashboard](assets/provider_manager_dashboard.png)
+
+#### Create Provider
+
+On opening the Provider Management, we are greeted with a screen with active providers and a **Floating Action Button**. Clicking on the Floating Action Button takes us to a new screen which asks for Provider details. The details are then matched with other existing Providers and a dialog box is shown to the user for confirmation.
```

Comment:
You are talking to the user, since this is the user guide. So change the wording e.g. 'Click the FAB, and fill out the provider details'. Make it clear and concise. Use **bold** or _italic_ to highlight where the user clicks in the app.